data/tutorials/lg_08_error_handling.md

```diff
@@ -73,34 +138,35 @@ Functions that can raise exceptions should be documented like this:
 <!-- $MDX skip -->
 ```ocaml
 val foo : a -> b
-(** foo does this and that, here is how it works, etc.
+(** [foo] does this and that, here is how it works, etc.
     @raise Invalid_argument if [a] doesn't satisfy ...
-    @raise Sys_error if filesystem is not happy *)
+    @raise Sys_error if filesystem is not happy
+*)
 ```
 
-### Stacktraces
+### Stack Traces
 
-To get a stacktrace when a unhandled exception makes your program crash, you
-need to compile the program in "debug" mode (with `-g` when calling
-`ocamlc`, or `-tag 'debug'` when calling `ocamlbuild`).
-Then:
+To get a stack trace when an unhandled exception makes your program crash, you
+need to compile the program in "debug" mode (with `-g` when calling `ocamlc`, or
+`-tag 'debug'` when calling `ocamlbuild`). Then:
 
 ```
 OCAMLRUNPARAM=b ./myprogram [args]
 ```
 
-And you will get a stacktrace. Alternatively, you can call, from within the program,
+And you will get a stack trace. Alternatively, you can call, from within the
+program,
 
 ```ocaml
```

Comment:
Is the code li 160 - 191 examples of code the user will write themselves, or is it something the reader should be able to test when going through this doc?